mempool/reactor_test.go

```diff
@@ -349,93 +349,93 @@ func TestMempoolFIFOWithParallelCheckTx(t *testing.T) {
 // Note: in this test we know which gossip connections are active or not because of how the p2p
 // functions are currently implemented, which affects the order in which peers are added to the
 // mempool reactor.
-func TestMempoolReactorMaxActiveOutboundConnections(t *testing.T) {
-	config := cfg.TestConfig()
-	config.Mempool.ExperimentalMaxGossipConnectionsToNonPersistentPeers = 1
-	reactors, _ := makeAndConnectReactors(config, 4)
-	defer func() {
-		for _, r := range reactors {
-			if err := r.Stop(); err != nil {
-				require.NoError(t, err)
-			}
-		}
-	}()
-	for _, r := range reactors {
-		for _, peer := range r.Switch.Peers().Copy() {
-			peer.Set(types.PeerStateKey, peerState{1})
-		}
-	}
-
-	// Add a bunch transactions to the first reactor.
-	txs := newUniqueTxs(100)
-	tryAddTxs(t, reactors[0], txs)
-
-	// Wait for all txs to be in the mempool of the second reactor; the other reactors should not
-	// receive any tx. (The second reactor only sends transactions to the first reactor.)
-	checkTxsInMempool(t, txs, reactors[1], 0)
-	for _, r := range reactors[2:] {
-		require.Zero(t, r.mempool.Size())
-	}
-
-	// Disconnect the second reactor from the first reactor.
-	firstPeer := reactors[0].Switch.Peers().Copy()[0]
-	reactors[0].Switch.StopPeerGracefully(firstPeer)
-
-	// Now the third reactor should start receiving transactions from the first reactor; the fourth
-	// reactor's mempool should still be empty.
-	checkTxsInMempool(t, txs, reactors[2], 0)
-	for _, r := range reactors[3:] {
-		require.Zero(t, r.mempool.Size())
-	}
-}
+// func TestMempoolReactorMaxActiveOutboundConnections(t *testing.T) {
```

Comment:
Why do we need to comment these tests out?

Reply:
They are failing almost constantly. I want to deal with them separately.

Reply:
see https://github.com/cometbft/cometbft/issues/3689